### ISSUE TITLE
Is the model sensitive to initialization ? 

### ISSUE THREAD

**Issue by pansanity666** [sxyu/pixel-nerf#45]
Hi,
In my experiments, I found that sometimes the  visualized images in  ./visuals becomes black images during early epochs, either coarse or fine output (seems to be collapsed?). I am not sure this is due to my modification of code, or the model itself is sensitive to the initialization weights?  I noticed that you saved the initial weight of model as 'backup'. Have you ever encountered this issue during your experiments ?

**Comment by GongXinyuu** [comment]
same here

**Comment by YoussefFathi** [comment]
same here, this happens a lot during my training runs which is quite frustrating.

**Comment by TianyuDai** [comment]
I got the same problem especially when training on images with large black background. I solved this problem by reducing the learning rate.

**Comment by jianglongye** [comment]
same here, sometimes i need to rerun experiments to make it work
